convert2rhel/checks.py

```diff
@@ -95,56 +97,59 @@ def check_convert2rhel_latest():
         if ver_compare > 0:
-            latest_version = package_version[1:]
-
-    # After the for loop latest_version will have the epoch ,version, and release ex:("0" "0.26" "1.el7") information from convert2rhel yum repo.
-    # the release for latest_verion will be "1.el7" and the relase for convert2rhel_version will be hard coded as "0" below,
-    # therefore when the versions are the same the latest_version's release field will cause it to evaluate as later
-    ver_compare = rpm.labelCompare(("0", convert2rhel_version, "0"), ("0", latest_version[1], "0"))
+            latest_available_version = package_version[1:]
+
+    # we hard code 0 for the epoch and release below because latest_verion will
+    # contain "1.el7" for the release but we don't know what the release for
+    # installed_convert2rhel_version will be, if we did not hard code to 0 latest_available_version
+    # will evaluate as later then installed_convert2rhel_version even when the versions
+    # were the same.
+    #
+    # ver_compare is set to
+    # * -1 if lastest_version is greater then installed_convert2rhel_version,
+    # * 0 if they are equal
+    # * 1 if installed_convert2rhel_version is greater the lastest_version
+    ver_compare = rpm.labelCompare(("0", installed_convert2rhel_version, "0"), ("0", latest_available_version[1], "0"))
     if ver_compare < 0:
         if "CONVERT2RHEL_UNSUPPORTED_VERSION" in os.environ:
             logger.warning(
                 "You are currently running %s and the latest version of convert2rhel is %s.\n"
-                "'CONVERT2RHEL_UNSUPPORTED_VERSION' environment detected, continuing conversion"
-                % (convert2rhel_version, latest_version[1])
+                "'CONVERT2RHEL_UNSUPPORTED_VERSION' environment variable detected, continuing conversion"
+                % (installed_convert2rhel_version, latest_available_version[1])
             )
 
         else:
             if int(system_info.version.major) <= 6:
                 logger.warning(
                     "You are currently running %s and the latest version of convert2rhel is %s.\n"
-                    "Only the latest version is supported for conversion." % (convert2rhel_version, latest_version[1])
+                    "We encurage you to update to the latest version"
+                    % (installed_convert2rhel_version, latest_available_version[1])
                 )
 
             else:
                 logger.critical(
                     "You are currently running %s and the latest version of convert2rhel is %s.\n"
                     "Only the latest version is supported for conversion. If you want to ignore"
-                    " this you can set 'CONVERT2RHEL_UNSUPPORTED_VERSION' to continue"
-                    % (convert2rhel_version, latest_version[1])
+                    " this check set the enviroment variable 'CONVERT2RHEL_UNSUPPORTED_VERSION=1' to continue."
```

Comment:
I don't know, but it seems to me that a `, then` is missing in that phrase. It looks strange without it. 

```suggestion
                    " this check, then set the environment variable 'CONVERT2RHEL_UNSUPPORTED_VERSION=1' to continue."
```

I'm not an expert in English, so feel free to deny this comment if it's wrong!

Reply:
you are definitely right about this "then" should be there!